src/passes/WasmIntrinsics.cpp.in

```diff
@@ -1,7 +1,7 @@
 #include "passes/intrinsics-module.h"
 
-static const char theModule[@WASM_INTRINSICS_SIZE@] = {
-@WASM_INTRINSICS_EMBED@
+static const char theModule[@WASM_INTRINSICS_SIZE_2@] = {
```

Comment:
Do you need the size here at all?  Won't it be implicit?

Reply:
Yes, we should be able to leave this out.